packages/kbn-optimizer/src/index.ts

```diff
@@ -14,4 +14,4 @@ export * from './node';
 export * from './limits';
 export * from './cli';
 export * from './report_optimizer_timings';
-export * from './babel_runtime_helpers';
+export * from './audit_bundle_dependencies';
```

Comment:
rename the dir because it no-longer contained only the logic for babel_runtime_helpers 😇